src/convertToObject.js

```diff
@@ -7,6 +7,16 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stringArr = sourceString
```

Comment:
The variable name `stringArr` could be improved to better describe its contents. Consider a name that reflects the nature of the data it holds, such as `styleDeclarations` or `stylePairs`.